src/options.go

```diff
@@ -650,7 +660,8 @@ func defaultOptions() *Options {
 		WalkerRoot:   []string{"."},
 		WalkerSkip:   []string{".git", "node_modules"},
 		Help:         false,
-		Version:      false}
+		Version:      false,
```

Comment:
Can you exclude unrelated style changes?

Reply:
Sry, sure. Editor auto-format. Reverted.